agent/mission_solver/data_loader.py

```
@@ -0,0 +1,158 @@
+
+
+def load_quest_enemies(region: str = "CN") -> dict[int, QuestPhase]:
+    """
+    加载本地副本敌人数据。
+
+    Returns:
+        {quest_id: QuestPhase} 字典
+    """
+    data = _load_json(f"quest_enemies_{region}.json")
+    quests = {}
+    for quest_id_str, quest_data in data.items():
```

Ruff (0.15.11)

[warning] Loop control variable `quest_id_str` not used within loop body

Rename unused `quest_id_str` to `_quest_id_str`

(B007)